Clinical-Genomics/cg
# Add scout support for tomte

## Description

### Added

- Scout config creation and upload for tomte

### Changed

- Updated delivery report for tomte to include information about which files are uploaded to Scout 


### How to prepare for test

    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b add_scout_for_tomte -a
    ```

### How to test

- Store case and generate report if not done already,`cg workflow tomte store legalpeacock; cg generate delivery-report legalpeacock -f` . Make sure delivery type is set to `scout`.
- [x] `cg upload -c legalpeacock -r` . Make sure the upload is successful
- [x] The scout_load.yaml should look like this:
```
---
analysis_date: 2024-08-07 10:05:18
default_gene_panels: []
delivery_report: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/delivery-report.html
family: legalpeacock
family_name: XXXX-RNA-5M_downsampled
gene_panels:
- BRAIN
- Cardiology
- CILM
- CH
- CTD
- DIAB
- ENDO
- EP
- HEARING
- HYDRO
- IBMFS
- IEM
- IF
- mcarta
- MHT
- MIT
- MOVE
- mtDNA
- NBS-M
- NEURODEG
- NMD
- OMIM-AUTO
- OPTIC
- PANELAPP-GREEN
- PEDHEP
- PID
- PIDCAD
- RETINA
- SKD
- SOVM
- STROKE
- AID
- Inherited cancer
human_genome_build: '38'
multiqc_rna: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/multiqc_report.html
omics_files:
  fraser: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/legalpeacock_fraser_top_hits_clinical.tsv
  outrider: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/legalpeacock_outrider_top_hits_clinical.tsv
owner: cust000
samples:
- analysis_type: wts
  father: '0'
  mother: '0'
  phenotype: unknown
  rna_alignment_path: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/ACC9480A3DS5C0M.cram
  rna_coverage_bigwig: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/ACC9480A3DS5C0M.bw
  sample_id: ACC9480A3DS5C0M
  sample_name: ACC9480A3DS5C0M
  sex: male
  splice_junctions_bed: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/ACC9480A3DS5C0M_junction.bed.gz
  tissue_type: unknown
track: rare
vcf_snv: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/legalpeacock_vep_clinical.vcf.gz
vcf_snv_research: /home/proj/stage/housekeeper-bundles/legalpeacock/2024-08-07/legalpeacock_vep_research.vcf.gz
```
- [x] Scout contains Clinical WTS outliers
<img width="988" alt="image" src="https://github.com/user-attachments/assets/bbf4979f-a546-4597-b8bd-04b322f4cf96">

- [x] Test scout upload for a case for which DROP was not run. scout_load.yaml should contain `omics_files: {}` and Scout should not show the outliers table.
<img width="668" alt="image" src="https://github.com/user-attachments/assets/060f1518-fc46-4e61-8fc0-c1ddeed16f5e">

- [x] Delivery report is updated to include information about files uploaded to Scout

## Review

Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a


## Implementation Plan
